security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc

```diff
@@ -18,7 +18,7 @@ authorized auditor to achieve compliance with a standard.
 
 [IMPORTANT]
 ====
-The Compliance Operator might report incorrect results on managed platforms, such as OpenShift Dedicated, Red Hat OpenShift Service on AWS, and Azure Red Hat OpenShift. For more information, see the link:https://access.redhat.com/solutions/6983418[Red Hat Knowledgebase Solution #6983418].
+The Compliance Operator might report incorrect results on some managed platforms, such as OpenShift Dedicated and Azure Red Hat OpenShift. For more information, see the link:https://access.redhat.com/solutions/6983418[Red Hat Knowledgebase Solution #6983418].
```

Comment:
Do we need to mention ROSA classic here? Thanks. 
Since it says "on some managed platforms, such as", it is good if you don't want to update here.
https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html